testnets/doravotatestnet/assetlist.json

```diff
@@ -19,7 +19,7 @@
         "display": "DORA",
         "symbol": "DORA",
         "logo_URIs": {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/cudostestnet/images/doravota.png"
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/doravotastestnet/images/doravota.png"
```

Comment:
This image doesn't exist. Could we point this to ...chain-registry/master/doravota/images/doarvotsa.png